apps/namadillo/src/App/Transfer/common.ts

```diff
@@ -34,7 +34,7 @@ export const isShieldedAddress = (address: string): boolean => {
 };
 
 export const isTransparentAddress = (address: string): boolean => {
-  return address.startsWith("tnam");
+  return address.startsWith("tnam") && address.length === 45;
```

Comment:
Just want to note that I'm adding this and want to double check that there's no caveats to this.

Reply:
LGTM